code/game/turfs/closed/minerals.dm

```diff
@@ -545,20 +545,26 @@
 		to_chat(usr, "<span class='warning'>Only a more advanced species could break a rock such as this one!</span>")
 		return FALSE
 	var/mob/living/carbon/human/H = user
-	if(H.mind.get_skill_level(/datum/skill/mining) >= SKILL_LEVEL_LEGENDARY)
+	if(H.mind.get_skill_level(/datum/skill/mining) >= SKILL_LEVEL_MASTER)
```

Comment:
this is supposed to be the ultimate feat so why not just keep it legendary tbf

Reply:
Noone is going to bring a pickaxe to make use of it anyway, so its not like it matters.

Reply:
>to_chat(usr, "<span class='warning'>The rock seems to be too strong to destroy. Maybe I can break it once I become a **master** miner.</span>"

owned